Documentation/BabylonCommands.md

```diff
@@ -0,0 +1,110 @@
+
+# 🕹 Commands implemented in the Babylon Bot
+
+The Stevenson instance used at Babylon, whose code can be found in the `App/` directory, declares the features described below, which mainly consist of 3 main core features:
+
+* Invoking a `fastlane` command on CI
+* Responding to a GitHub comment to trigger a CI pipeline
+* [Generating the CRP ticket](CRP.md)
+
+## `/stevenson`
+
+This is the main Slack command, used to namespace other commands. It's implemented in `MainCommand.swift`
+
+This command expects one of the following sub-commands, which are described in more details below:
+
+* `/stevenson fastlane …`
+* `/stevenson appcenter …`
+* `/stevenson testflight …`
+* `/stevenson crp …`
+
+## `/stevenson fastlane`
+
+This and the 2 following commands are implemented in `SlackCommand+Fastlane.swift`
+> Also still accessible via `/fastlane` directly, though deprecated in favor of being invoked as a `/stevenson` subcommand instead.
+
+This command allows you to trigger a CircleCI job which will run the provided lane.
+
+```
+/stevenson fastlane <lane> [branch:<branch>] [target:<target>] [<other_options>]
+```
+
+* If `branch:` is not provided, defaults to running the lane on `develop`
+* `target:` is only used by some lanes in Fastlane, like `appcenter` or `testflight`. In most cases it's not needed
+* Any potential `<other_options>` depend on each lane. Check our Fastlane to see which are available for each. Examples include `version:` for `appcenter` and `testflight` lanes
+
+
+## `/stevenson appcenter`
+
+> Also still accessible via `/appcenter` directly, though deprecated in favor of being invoked as a `/stevenson` subcommand instead.
+
+
+```
+/stevenson appcenter <target> [version:<version>] [branch:<branch>]
+```
+
+* This command is just a convenience doing the same as `/stevenson fastlane appcenter target:<target> version:<version> branch:<branch>`
+* If `branch:` is not provided, defaults to `release/<target>/<version>`
+
+## `/stevenson testflight`
+
+> Also still accessible via `/testflight` directly, though deprecated in favor of being invoked as a `/stevenson` subcommand instead.
+
+```
+/stevenson testflight <target> [version:<version>] [branch:<branch>]
+```
+
+* This command is just a convenience doing the same as `/stevenson fastlane testflight target:<target> version:<version> branch:<branch>`
+* If `branch:` is not provided, defaults to `release/<target>/<version>`
+
+
+## `/stevenson crp`
+
+> Still accessible via `/crp` directly, though deprecated in favor of new name.
+
+This command creates a "CRP Ticket" in our JIRA, which is a JIRA ticket gathering all the tickets that will be part of a release to the Stores in order to validate approval for that release. The creation and approval of this ticket to make a new release is part of our SSDLC. 
+
+For more information about the CRP, visit [the dedicated page in your Playbook](https://github.com/babylonhealth/ios-playbook/blob/master/Cookbook/Technical-Documents/CRP-Bot.md).
```

Comment:
PR pending for this future document: https://github.com/babylonhealth/ios-playbook/pull/313
(So not yet available in master… until the PR in playbook is merged)